Code/Engine/Foundation/Math/Implementation/Easing_inl.h

```diff
@@ -0,0 +1,268 @@
+#pragma once
+
+#include <Foundation/Math/Math.h>
+
+namespace ezMath
+{
+
+  template <typename Type>
+  EZ_ALWAYS_INLINE Type EaseInLinear(Type t)
```

Comment:
Since you are taking function pointers to these, inlining might not be possible.
But for your use case it's not critical anyway. Just an observation.

Reply:
Removed the function pointer altogether, It is more useful to have the function return the value of the easing function type.

Reply:
Yes, very good, I prefer it that way as well.